include/http/request_handler.js

```
@@ -1149,15 +1151,17 @@ module.exports = function RequestHandlerModule(pb) {
      * @param {Object} route
      * @param {Object} route.path_vars
      */
-    RequestHandler.prototype.getPathVariables = function(route) {
+    RequestHandler.prototype.getPathVariables = function(route, method) {
         var pathVars = {};
         var pathParts = this.url.pathname.split('/');
-        Object.keys(route.path_vars).forEach(function(field) {
-            pathVars[field] = pathParts[route.path_vars[field]];
-        });
+        let methodKey = !route.path_vars[method] && route.path_vars['ALL'] ? 'ALL' : method;
```

nvm, we're before render so we can throw an error here